rotkehlchen/tests/fixtures/blockchain.py

```diff
@@ -222,8 +221,8 @@ def _make_substrate_manager(
         if chain_type == SubstrateChain.KUSAMA:
             substrate_manager.chain_properties = SubstrateChainProperties(
                 ss58_format=KUSAMA_SS58_FORMAT,
-                token=KUSAMA_TOKEN,
-                token_decimals=KUSAMA_TOKEN_DECIMALS,
+                token=A_KSM,
```

Comment:
I think this would fail, no? I guess it only takes the symbol as we said.

Reply:
I've checked and it seems we assume is the asset always